pkg/metrics/metrics.go

```diff
@@ -186,12 +186,12 @@ func (r *standardRegistry) ServiceServerUpGauge() metrics.Gauge {
 	return r.serviceServerUpGauge
 }
 
-// ScalableHistogram is a Histogram with a predefined time unit used when
-// producing observations allowing the call to Start() and ObserveDuration()
-// without explicitly setting the observed value.
+// ScalableHistogram is a Histogram with a predefined time unit, used when
+// producing observations without explicitly setting the observed value.
 type ScalableHistogram interface {
 	With(labelValues ...string) ScalableHistogram
 	Start()
+	StartAt(t time.Time)
```

Comment:
I'm not sure it's interesting to provide this exported StartAt. It is only a very small convenience over Start, right? there is nothing that StartAt allows that could not also be done just with Start, isn't it?
If you want to get rid of Start, and keep only StartAt, that's fine imo, but I think keeping both is overkill.

Reply:
agreed, I'll keep just StartAt